docs/_snippets/getting-started/use-builder.html

```diff
@@ -0,0 +1,73 @@
+<div class="builder-cta">
+	<h2 id="ckeditor-5-builder">CKEditor&nbsp;5 Builder</h2>
+
+	<p>
+		In our interactive Builder you can quickly get a taste of CKEditor&nbsp;5. It offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs. You can easily select:
+	</p>
+
+	<ul>
+		<li>The editor type.</li>
+		<li>The features you need.</li>
+		<li>Preferred framework (React, Angular, Vue or Vanilla JS).</li>
+		<li>Preferred distribution method.</li>
+	</ul>
+
+	<p>At the end you get ready-to-use code tailored to your needs!</p>
+
+	<p>
+		<a href="https://ckeditor.com/ckeditor-5/builder?redirect=docs" target="_blank">Check out our interactive Builder</a>
+	</p>
+</div>
+
+<style>
+	.builder-cta {
+		border: 1px solid #ECE6F7;
+		border-left: 4px solid #A183D9;
+		padding: 24px;
+		margin: 32px 0 20px;
+	}
+
+	.builder-cta h2 {
+		padding-top: 0;
+		border: 0;
+		font-weight: bold;
```

Comment:
I think the heading stands out too much. How about unifying it with other headings?
```suggestion
		font-weight: 400;
```